plugins/registry.js

```diff
@@ -872,4 +877,19 @@ export default class PluginRegistry {
 
         return id;
     }
+
+    // INTERNAL: Subject to change without notice.
+    // Register a handler to retrieve stats that will be displayed on the system console
+    // Accepts the following:
+    // - handler - Func to be called to retrieve the stats from plugin api
```

Comment:
Maybe state that the handler needs to return `Record<string, PluginAnalyticsRow>`